files/pi-mpi-hpc.txt

```diff
@@ -0,0 +1,9 @@
+1,512000000,5859.375000,28.546736,0.003635
```

Comment:
- These `pi-*.txt` files represent CSV data. Please change their extensions and add header rows.
- Are these files to be used in the lesson? If not, consider bundling the results, and plot generator scripts, in a tarball.